scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala

```diff
@@ -17,6 +17,7 @@
 
 package com.spotify.scio
 
+import com.google.bigtable.repackaged.com.google.cloud.grpc.BigtableClusterUtilities
```

Comment:
Its in the release notes so it must be intended for public use. It probably is just shaded in the hbase-dataflow-connector for some reason https://github.com/GoogleCloudPlatform/cloud-bigtable-client/releases